docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md

```diff
@@ -94,14 +93,14 @@ It's very likely that you've already installed AdGuard VPN before.
 
 - Check if AdGuard VPN is already installed on your computer. You can do it by pressing the *Win* key and typing in ***AdGuard VPN***.
 
-- Maybe there's some leftover files from a previous AdGuard VPN installation. Uninstall AdGuard using our special [uninstall tool](../../installation#advanced) and then repeat the installation.
+- Maybe there are some leftover files from a previous AdGuard VPN installation. Uninstall AdGuard using our special [uninstall tool](../../installation#advanced) and then repeat the installation.
 
 ### Other errors {#other}
 
 If you've encountered an error that's not listed above, it is possible that we can solve it by ourselves. But in order to do that, we need log files from you. Please perform the following steps:
```

Comment:
you have, that is

Reply:
please check the instances with `'`

Reply:
Changed to "you have", but kept "that's" as it seems more natural and easy to read that way